src/main/java/io/mosip/digitalcard/service/impl/PDFCardServiceImpl.java

```diff
@@ -159,6 +162,9 @@ public byte[] generateCard(org.json.JSONObject decryptedCredentialJson, String c
 				attributes.put("isPhotoSet",isPhotoSet);
 			}
 			uin = decryptedCredentialJson.getString("UIN");
+			if(additionalAttributes.containsKey("template")) {
```

Comment:
I think this is template-code only not the template itself, so, let's rename the attribute name something like `templateCode` instead of `template`